node/src/components/storage.rs

```diff
@@ -338,9 +338,7 @@ impl Storage {
                         HashingAlgorithmVersion::V1 => {
                             let _ = deleted_block_body_hashes_v1.insert(*block_header.body_hash());
                         }
-                        HashingAlgorithmVersion::V2 => {
-                            let _ = deleted_block_body_hashes_v2.insert(*block_header.body_hash());
-                        }
+                        HashingAlgorithmVersion::V2 => {}
```

Comment:
Maybe consider `if let` instead of match with empty arm?

Reply:
I think I'll actually use this arm again, but in a slightly different way.